### ISSUE TITLE
Run tests thru babel when babel 7 arrives

### ISSUE THREAD

**Issue by skellock** [skellock/typescript-with-electron-react-kit#57]
When `babel 7` arrives, we'll be able to switch over to that toolchain and use `ava` a little nicer.

Currently we're compiling all `ts` to `js` and running ava from there.  `babel 7` will have support for `typescript`.  🎉

**Comment by rmevans9** [comment]
WHAT! Babel 7 supports TypeScript? That is A-M-A-Z-I-N-G :-D

In unrelated news: That should make the TypeScript integration with React Native easier.

**Comment by skellock** [comment]
Ya!  Cool eh?!

`typescript` is really simple to integrate with React Native because of the new transformer features that shipped in `0.45`!

Just need to install [`react-native-typescript-transformer`](https://github.com/ds300/react-native-typescript-transformer).

And add this `rn-cli.config.js` file to the root of your project:

```js
module.exports = {
  getTransformModulePath () {
    return require.resolve('react-native-typescript-transformer')
  },
  getSourceExts () {
    return ['ts', 'tsx']
  }
}
```

**Comment by skellock** [comment]
So, the typescript support in Babel only extends to stripping annotations and not actually performing type checking. Nothing will likely change with this setup.
